mmv1/third_party/terraform/website/docs/r/storage_bucket_object.html.markdown

```diff
@@ -69,6 +69,8 @@ One of the following is required:
 * `customer_encryption` - (Optional) Enables object encryption with Customer-Supplied Encryption Key (CSEK). [Google [documentation about](#nested_customer_encryption) CSEK.](https://cloud.google.com/storage/docs/encryption/customer-supplied-keys)
     Structure is documented below.
 
+* `retention` - (Optional) The [object retention](http://cloud.google.com/storage/docs/object-lock) settings for the object. The retention settings allow an object to be retained until a provided date. Structure is documented below.
```

Comment:
This should link to the nested block - `customer_encryption` should as well but that's a separate issue. Compare to blocks on https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/alloydb_instance

```suggestion
* `retention` - (Optional) The [object retention](http://cloud.google.com/storage/docs/object-lock) settings for the object. The retention settings allow an object to be retained until a provided date. Structure is [documented below](#nested_retention).
```

Reply:
Done (also fixed `customer_encryption`)